dep0/app/dep0/main.cpp

```diff
@@ -105,18 +126,28 @@ int main(int argc, char** argv)
             llvm::WithColor::note(llvm::outs(), f) << "typechecks correctly" << '\n';
             continue;
         }
-        auto transform_result =
-            dep0::transform::run(
-                *typechecked_module,
-                dep0::transform::beta_delta_normalization_t{}
-            );
-        if (not transform_result)
+        if (not skip_transformations)
         {
-            std::ostringstream str;
-            dep0::pretty_print(str, transform_result.error());
-            llvm::WithColor::error(llvm::errs(), f) << "transformation error: " << str.str() << '\n';
+            auto const transform_result =
+                dep0::transform::run(
+                    *typechecked_module,
+                    dep0::transform::beta_delta_normalization_t{}
+                );
+            if (not transform_result)
+            {
+                std::ostringstream str;
+                dep0::pretty_print(str, transform_result.error());
+                llvm::WithColor::error(llvm::errs(), f) << "transformation error: " << str.str() << '\n';
+            }
         }
-        auto llvm_module = dep0::llvmgen::gen(llvm_context, f, *typechecked_module);
+        if (print_ast)
+            dep0::ast::pretty_print(std::cout, *typechecked_module) << std::endl;
```

Comment:
Should be possible to do better, eg by writing to a file (with default name `{input-file}.out.depc` and user can always pass `/dev/stdout` if they want to see on screen

Reply:
meh whatever; I think eventually I will revamp the command line arguments altogether